openmc/statepoint.py

```diff
@@ -699,8 +736,10 @@ def link_with_summary(self, summary):
 
         Raises
         ------
-             ValueError : An error when the argument passed to the 'summary' 
```

Comment:
I think an important detail that was missing in my original comment above is that it is necessary to link against a summary to populate the ``Tallies`` with any user-specified "name" XML tags, which can provide an easy and effective way to call ``get_tally(name='....')`` above